content/en/docs/Installing and Configuring DreamFactory/_index.md

```diff
@@ -211,6 +211,14 @@ Immediately following this prompt you'll be informed of successful setup:
 	* your instance using a browser. You can run "php artisan serve"
 	* to try out your instance without setting up a web server.
 	**********************************************************
+After creating an Admin, your instance may show the following error:
+```
+Admins
+There is no root administrator selected. Some functionality might not work. Use df:root_admin command to choose one.
+```
+
+A very simple approach to solve it, is to reset the root from inside your container:
+`$ php artisan df:root_admin`
```

Comment:
Great! Thanks much. I think this will be better a little further down, perhaps to line 231 (just before the Introducing the .env File h2 header.

The error will occur when upgrading to a gold license, also I think container is quite specific language to docker, so I suggest making the language a little more general. As below:

Finally I think it would be good to put this in an alert. How about we wrap what you put inside this (also my suggested changes):
```
{{< alert title="Note">}}
When upgrading to our Gold edition your instance may show the following error upon login:

Admins
There is no root administrator selected. Some functionality might not work. Use df:root_admin command to choose one.

A very simple approach to solve it, is to reset the root administrator by running the following command:
`$ php artisan df:root_admin`
(If you have multiple administrators, it will ask you for who you would like to make the root admin)
{{< /alert >}} 
```